talkkkas.yml

```diff
@@ -0,0 +1,4 @@
+{
+"dfg":"sdgf",
+"awsTokens":"sdrfgilusge3425345"
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:** "awsTokens":"******************"  
[This comment was created by Aqua Pipeline]